### DISCUSSION TITLE
Could I publish message with wildcards

### DISCUSSION THREAD

**Discussion by fy2462** [nats-io/nats-server#2855] [Q&A] [answered]
Could I use some wildcards to broadcast message?

`publish_topic: "abc.*.cd.>" ?`

I found only subjects can use wildcards, right?

Thanks.

**Comment by derekcollison** [comment, marked as answer]
You can only send to literal subjects when publishing. Wildcards are only applicable for subscriptions.

**Reply by fy2462** [reply]
Got it, thanks.
